app/templates/pages/department.html

```diff
@@ -123,6 +124,16 @@ <h4>Thiemo Kreuz<br>
                         <div class="col-lg-12">
                             <h3>Wikidata Team</h3>
                         </div>
+                        <div class="col-lg-4 col-md-6 text-center">
+                            <div class="thumbnail">
+                                <img class="img-responsive" src="https://wikimedia.de/img/staff/default.jpg" alt="">
+                                <div class="caption">
+                                    <h4>Michael Große<br>
+                                        <small>Software Developer</small>
+                                    </h4>
+                                </div>
```

Comment:
Feel free to add my wikimedia email and github here as well:

```suggestion
                                    <ul class="list-inline">
                                        <li><a href="mailto:michael.grosse@wikimedia.de"><i class="fa fa-2x fa-envelope-square"></i></a></li>
                                        <li><a href="https://github.com/micgro42"><i class="fa fa-2x fa-github-square"></i></a></li>
                                    </ul>
                                </div>
```

Reply:
(we can do this in another pr as well, when we are adding the pictures)

Reply:
I'm adding way more people :D We can do that later too.